client/src/components/Menu/index.tsx

```diff
@@ -20,6 +20,12 @@ import {
     addNextResults,
 } from '../../store/search/actions';
 
+const search_hashtags = [
```

Comment:
I believe it will change soon. Then TODO needs?

Reply:
Yes, it will change. I added TODO